### PR TITLE
Add stickers promo card

### PR DESCRIPTION
**Pull request by MelSmith104**
Screen Shot 2022-01-17 at 6 42 04 PM

still wip - need to edit the copy and make slight changes to airtable form

### CODE REVIEW

**Comment by kunalbotla** [comment]
yay! this looks awesome!

**Comment by maxwofford** [comment]
@eilla1 could you help add some stickers on top that pop out or something like that to add to the visuals?

File: app/views/events/promotions.html.erb [hunk 1]
```diff
@@ -145,6 +145,26 @@
   </p>
 </div>
 
+<div class="card mb3" id="stickers">
+  <div class="card__banner card__banner--top flex justify-between items-center promo-stickers">
+    <h3 class="h1 white p2 my0">Stickers</h3>
+    <%= link_to 'Request',
```
Comment (maxwofford) [inline app/views/events/promotions.html.erb]: "Request" feels a little out of place here. Maybe "apply" to keep it consistent with the other promos?

File: app/assets/stylesheets/components/_promos.scss [hunk 2]
```diff
@@ -50,10 +55,15 @@
...
-   }
- }
\ No newline at end of file
+  .promo-domains {
+    background-color: #363a44;
+    color: white;
+  }
+  .promo-stickers {
+   background-color: #343a44;
+   border: 2px solid #EC3750;
```
Comment (maxwofford) [inline app/assets/stylesheets/components/_promos.scss]: Try cycling light/dark mode and check what the 2px border does to the size of the div:

https://user-images.githubusercontent.com/5891442/149857836-7df67783-b374-4c14-9488-6a153671cafb.mov

File: app/views/events/promotions.html.erb [hunk 3]
```diff
@@ -145,6 +145,26 @@
   </p>
 </div>
 
+<div class="card mb3" id="stickers">
+  <div class="card__banner card__banner--top flex justify-between items-center promo-stickers">
+    <h3 class="h1 white p2 my0">Stickers</h3>
```
Comment (maxwofford) [inline app/views/events/promotions.html.erb]: Seems like other promos have the sponsor's logo in the top bar. Could we add our own here?


This is badly done, but ex:
Screen Shot 2022-01-17 at 20 58 26

**Comment by garyhtou** [comment]
wahoo this is amazing!!

File: app/views/events/promotions.html.erb [hunk 4]
```diff
@@ -145,6 +145,26 @@
...
+    <%= link_to 'Request',
+      organizer_signed_in? ? "https://hack.af/hcb-stickers?prefill_Recipient%20Name=#{current_user.full_name}&prefill_Login%20Email=#{current_user.email}" : root_path,
+      class: "btn bg-success mr1 #{'disabled' unless organizer_signed_in?}",
+      target: :_blank
+     %>
+  </div>
+
+  <div class="mb0">
+    <p><strong>Want stickers?</strong> You can request a Hack Club Bank sticker envelope for yourself or your teammates! Every shipment request is manually sent by the Bank team, and every sticker shipment is personally prepared, packed, and mailed, with the address hand written.</p>
```
Comment (maxwofford) [inline app/views/events/promotions.html.erb]: Screen Shot 2022-01-17 at 21 11 47

Here's a quick pass on copy. I grabbed the muted bottom section thing from the PVSA promo. Thoughts?

File: app/views/events/promotions.html.erb (hunk 3 above)
Comment (maxwofford) [reply app/views/events/promotions.html.erb]: Also, the title is a ambiguous. Is this hack club stickers? bank stickers? random tech stickers? Idk if we should just drop the words in the title & show some stickers, or if we want to "Stickers" -> "HCB Stickers", but let's fix this

File: app/views/events/promotions.html.erb [hunk 5]
```diff
@@ -145,6 +145,26 @@
   </p>
 </div>
 
+<div class="card mb3" id="stickers">
+  <div class="card__banner card__banner--top flex justify-between items-center promo-stickers">
+    <h3 class="h1 white p2 my0">Stickers</h3>
+    <%= link_to 'Request',
+      organizer_signed_in? ? "https://hack.af/hcb-stickers?prefill_Recipient%20Name=#{current_user.full_name}&prefill_Login%20Email=#{current_user.email}" : root_path,
```
Comment (maxwofford) [inline app/views/events/promotions.html.erb]: Can we auto-fill this address?

**Review by maxwofford — COMMENTED** [review]
It's looking great! I added a couple review comments based on opening the preview build and going through the flow, but it's really feeling close.

Outside the comments, I wanted to ask who this promo is going to be for. Do we ship outside the US?

File: app/views/events/promotions.html.erb (hunk 5 above)
Comment (exu3) [reply app/views/events/promotions.html.erb]: I don't think we have that information for users.

People submit their addresses through Airtable when they apply to run on Bank, but that information isn't requested when they first log into their account on bank.hackclub.com

**Comment by exu3** [comment]
Have addressed the above comments:

https://user-images.githubusercontent.com/72365100/149922680-b72fda4b-2fa0-4310-aa28-5fd008ab9209.mov

**Pull request merged**